libraries/AP_InertialSensor/AP_InertialSensor.cpp

```diff
@@ -453,6 +453,14 @@ const AP_Param::GroupInfo AP_InertialSensor::var_info[] = {
     // @Bitmask: 0:FirstIMU,1:SecondIMU,2:ThirdIMU
     AP_GROUPINFO("ENABLE_MASK",  40, AP_InertialSensor, _enable_mask, 0x7F),
 
+    // @Param: GYRO_FILTER2
+    // @DisplayName: Second Gyro filter cutoff frequency
+    // @Description: Filter cutoff frequency for gyroscopes. This can be set to a lower value to try to cope with very high vibration levels in aircraft. This option takes effect on the next reboot. A value of zero means track the value of GYRO_FILTER
```

Comment:
please add the needs_reboot atribute.

Reply:
I don't think it needs this does it? The code has been written to detect changes on the fly and update the filter. The same is true for INS_GYRO_FILT

Reply:
If that is the case then update the description text

Reply:
Ah good catch